cmd/oci-catalog/src/providers/dockerhub.rs

```diff
@@ -44,7 +60,7 @@ impl OCICatalogSender for DockerHubAPI {
         tx: mpsc::Sender<Result<Repository, Status>>,
         request: &ListRepositoriesRequest,
     ) {
-        let mut url = url_for_request(request);
+        let mut url = url_for_request_repositories(request);
 
         let client = reqwest::Client::builder().build().unwrap();
```

Comment:
Not now, but given the DockerHub rate limits, perhaps it is worthwhile adding sth like: https://crates.io/crates/governor for better handling it on the client side.

Reply:
Yeah, that'd be great to add in the future if we find we're hitting rate limits (I'll be testing this using the bitnami catalog, which is probably on the larger side of OCI helm registries).